libs/model/src/contest/Contests.projection.ts

```diff
@@ -210,8 +211,8 @@ async function updateScore(contest_address: string, contest_id: number) {
       prize:
         i < Number(details.payout_structure.length)
           ? (
-              (BigInt(prizePool) * BigInt(details.payout_structure[i])) /
-              100n
+              (Number(prizePool) * Number(details.payout_structure[i])) /
+              100
```

Comment:
Wondering if this will be a problem with larger values. Whenever I use BigInt, there's an error during some operation.